draft-ietf-oauth-transaction-tokens.md

```diff
@@ -542,7 +542,7 @@ Txn-Tokens SHOULD NOT be logged if they contain Personally Identifiable Informat
 
 # IANA Considerations {#IANA}
 
-This specification registers the following claims defined in Section {{txn-token-header}} to the OAuth Access Token Types Registry defined in {{RFC6749}}, and the following claims defined in Section {{txn-token-claims}} in the IANA JSON Web Token Claims Registry defined in {{RFC7519}}
+This specification registers the following field defined in Section {{txn-token-header}} to the OAuth Access Token Types Registry defined in {{RFC6749}}, and the following claims defined in Section {{txn-token-claims}} in the IANA JSON Web Token Claims Registry defined in {{RFC7519}}
```

Comment:
Lots to unpack here that goes beyond changing one word...

The referenced section `{{txn-token-header}}` is [JWT Header](https://www.ietf.org/archive/id/draft-ietf-oauth-transaction-tokens-01.html#name-jwt-header) which calls headers claims. Headers are not claims. Also the value of the `typ` header is supposed to be a media type so the `txn_token` value probably isn't quite right and the registration for it below certainly isn't right. Also the `urn:ieft:params:oauth:token-type:txn-token` URI is missing a registration and the [Txn-Token Request](https://www.ietf.org/archive/id/draft-ietf-oauth-transaction-tokens-01.html#name-txn-token-request) section that defines it has some major formatting issues. 

Some related reading, issues, and examples of these things being done in other specs follows: 

https://github.com/oauth-wg/oauth-transaction-tokens/issues/84
https://www.rfc-editor.org/rfc/rfc7515#section-4.1.9
https://www.rfc-editor.org/rfc/rfc7519#section-5.1
https://www.rfc-editor.org/rfc/rfc8725.html#name-use-explicit-typing
https://datatracker.ietf.org/doc/html/rfc9068#name-header
https://www.rfc-editor.org/rfc/rfc9449.html#name-dpop-proof-jwt-syntax
https://www.rfc-editor.org/rfc/rfc7519.html#section-10.2

Reply:
I'm pulling out the misuse of the "typ" Header Parameter as a separate issue so that it can be discussed on its own.

Reply:
https://github.com/oauth-wg/oauth-transaction-tokens/issues/97